src/main/com/example009/radek/v3/FileService.java

```diff
@@ -0,0 +1,93 @@
+package main.com.example009.radek.v3;
+
+import java.io.*;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileService {
+
+    final private String path = "src/main/com/example009/radek/v3/calculator-results/";
```

Comment:
Špatné pořadí klíčových slov. ... zjistí si jak to má správně být pokud ma promenne nasledujici klicove identifikatory a to pak dodrzuj 
final 
private 
static

Reply:
Přehozeno, díky za připomenutí